pkg/api/types.go

```diff
@@ -584,10 +584,36 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// If nothing exists at the given path, an empty directory will be created there.
+	HostPathUnset HostPathType = "DirectoryOrCreate"
+	// A directory entry of any type must exist at the given path
+	HostPathExists HostPathType = "Exists"
+	// A file must exist at the given path
+	HostPathFile HostPathType = "File"
+	// A block or character device must exist at the given path
+	HostPathDevice HostPathType = "Device"
```

Comment:
I think we care about the difference between block and char, and I can't imagine why anyone would ever say "I don't care which one".  They are very not the same.  

I.e. get rid of this one.